examples/isomorphic-flux-chat/js/main.js

```diff
@@ -0,0 +1,25 @@
+var mockData = require('./mock-data')
+var Nuclear = require('nuclear-js')
+var reactor = new Nuclear.Reactor({
+  debug: true,
+})
+window.reactor = reactor
+var Chat = require('./modules/chat')
+
+var ChatApp = require('./components/ChatApp.react')
+ChatApp = Nuclear.provideReactor(ChatApp)
```

Comment:
withContext is deprecated, now the child has to declare a contextType and parent has to declare childContextType. provideReactor is here to ... provide the reactor (by using childContextTypes) and children either have to use nuclearComponent or nuclearMixin.

This example is outdated ... check out the other universal PR, and the ref code is here https://github.com/jordangarcia/nuclear-js-react-addons/blob/master/nuclearComponent.js#L63